functests/utils/mcps/mcps.go

```diff
@@ -89,6 +89,18 @@ func GetConditionStatus(mcpName string, conditionType machineconfigv1.MachineCon
 	return corev1.ConditionUnknown
 }
 
+// GetConditionReason return the message of the given MCP
+func GetConditionReason(mcpName string, conditionType machineconfigv1.MachineConfigPoolConditionType) string {
+	mcp, err := GetByName(mcpName)
+	ExpectWithOffset(1, err).ToNot(HaveOccurred(), "Failed getting MCP by name")
+	for _, condition := range mcp.Status.Conditions {
+		if condition.Type == conditionType {
+			return condition.Reason
+		}
+	}
+	return "No reason found."
```

Comment:
please do not use some random strings. Either just return an empty string, or an error.

Reply:
Noted. This func has been removed.